views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/item.css

```diff
@@ -294,3 +294,18 @@ prm-location-items md-list-item {
   letter-spacing: 0.16px !important;
   font-weight: normal !important;
 }
+
+/* Ethical Description note: U/Body/Caption */
+ethical-description-note div {
+  font-family: proxima-nova;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 160%;
+  letter-spacing: 0.16px;
+  text-align: center;
```

Comment:
@ztucker4 I think the centered text does not work well for this longer message.  What I see (Firefox/Windows) is a multi-line message with each line centered independently.  This probably would look better with a block of text, with each line left-aligned to the others in the block (which could then be centered, probably).

This is what I see:
![primo_redesign_ethical](https://github.com/UCLALibrary/primo_ve/assets/2213836/b47758d2-0ecc-4643-81e9-ad53482d7c2d)